src/etc/inc/openvpn.inc

```diff
@@ -1096,8 +1106,32 @@ function openvpn_reconfigure($mode, $settings) {
 					$tls_directive = "tls-auth";
 					if ($mode == "server") {
 						$tlsopt = 0;
+						//switch($settings['tlsauth_keydir']){
```

Comment:
I'd prefer to see the server page implemented completely before this is imported rather than doing it halfway.

We can't automatically correlate client/server, that will have to be left to the end user to handle. I wouldn't worry about that case at all. So long as the defaults preserve the existing behavior it should be safe.